apps/dashboard/src/components/primitives/editor.tsx

```diff
@@ -0,0 +1,64 @@
+import { useEffect, useRef } from 'react';
+import { useCodeMirror, EditorView } from '@uiw/react-codemirror';
+import { LiquidHTML } from 'codemirror-lang-liquid';
+import { cva, VariantProps } from 'class-variance-authority';
+import createTheme from '@uiw/codemirror-themes';
+
+const editorVariants = cva('-mx-1 -mt-[2px] h-full w-full flex-1 [&_.cm-focused]:outline-none', {
+  variants: {
+    size: {
+      default: 'text-xs [&_.cm-editor]:py-1',
+      md: 'text-sm [&_.cm-editor]:py-2',
+    },
+  },
+  defaultVariants: {
+    size: 'default',
+  },
+});
```

Comment:
these styles adjust the editor to look good when placed inside the `InputField` component; basically to have the correct padding from the top